docs/how-to-configure-new-repository.md

```diff
@@ -30,13 +30,10 @@ for the repository.
    PRs. See the [Policies](#policies) section that suggest to ensure that `Require
    review from Code Owners` is checked.
 4. The team `foo-approvers` has `Write` permissions for the repository.
-5. The team `foo-maintainers` has `Maintain` permissions for the
+5. The team `foo-maintainers` has `Admin` permissions for the
    repository.
 6. Root-level `CODEOWNERS` file on the repository should include superset of
    people from both `foo-approvers` and `foo-maintainers`.
-7. Some repositories may include more individuals with `Admin` permissions.
```

Comment:
are there examples where this privilege is presently in use / necessary?

Reply:
Giving maintainers admin privileges removes the need for this doesn't it?

Reply:
I would think that retaining this flexibility, even if rarely used, might be good.  Does removing this mean that it MUST NOT be done in the future, or does it become undefined whether such exceptions can be made?

Reply:
I'm totally ok removing it now. Going forward if we will start developing more automation, this or similar item may be needed back. I think ultimate goal is to make maintainers life easier and if it will require "external" help, this should be fine.